lib/BloqadeSchema/Project.toml

```diff
@@ -11,6 +11,8 @@ Configurations = "5218b696-f38b-4ac9-8b61-a12ec717816d"
 GarishPrint = "b0ab02a7-8576-43f7-aa76-eaa7c3897c54"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
move this to test deps tho, if it's only for tests

Reply:
Does that go into [extras]?